src/main/java/team/five/amazonreviewutil/service/CsvParserService.java

```diff
@@ -0,0 +1,82 @@
+package team.five.amazonreviewutil.service;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import team.five.amazonreviewutil.entity.Review;
+import team.five.amazonreviewutil.repositiry.ReviewRepository;
+
+@Service
+public class CsvParserService {
+
+    @Autowired
+    ReviewRepository reviewRepository;
+
+    private static Logger logger = Logger.getLogger(CsvParserService.class);
+
+    private final int fieldsCount = 10;
+    private int itemsRead;
+    private final String fieldSeparator = ",";
+    Charset charset = StandardCharsets.UTF_8;
+    private final char lineBreak = '\n';
+
+    public void processInputFile(String fileName) throws IOException {
```

Comment:
@TatianaPirch Isn't it too much for one method??? Did you consider to separate the functionality?